topbar.js

```diff
@@ -799,16 +806,34 @@ function fixTopBar() {
     }
 }
 
+function fixWorkspaceIndicator() {
+    prefs.show_workspace_indicator ? menu.show() : menu.hide();
+    Tiling.spaces.forEach(s => s.showWorkspaceIndicator());
+}
+
 function fixFocusModeIcon() {
     prefs.show_focus_mode_icon ? focusButton.show() : focusButton.hide();
     Tiling.spaces.forEach(s => s.showFocusModeIcon());
 }
 
+/**
+ * Sets the space focusModeIcon position based off the position
+ * of the focusButton (mimics it).
+ * @param {Number} timeout needed to allow update before querying position.
+ */
+function setSpaceFocusModeIconPositions(timeout=0) {
```

Comment:
Do we need a specific timeout here? Or does `0` always work?

Reply:
default is 0, but a small timeout is needed when workspace indicator emits `show` and `hide` signals.

See https://github.com/paperwm/PaperWM/pull/513/files#diff-b0cab10735fabfe8a6e4455997859f505dbbe12a813ba3227c2952ba5a89ecddR671-R672

Reply:
Yes, `0` always works, hence set it as default.

We also could so something like

```
function setSpaceFocusModeIconPositions(timeout) {
    timeout = timeout ?? 0;
    ...
}
```
Which is kind of equivalent (but maybe preferred since it would also cover the case where a developer might do a `setSpaceFocusModeIconPositions(undefined)` or `setSpaceFocusModeIconPositions(null)`) - I can change that one.